src/lib/utils/twMergeConfig.tsx

```diff
@@ -0,0 +1,12 @@
+import {extendTailwindMerge} from "tailwind-merge"
+
+const customTwMerge = extendTailwindMerge({
+  extend: {
+    classGroups: {
+      // Creates an array containing type-# classes from 0 to 1
+      "font-size": Array.from({length: 10}, (_, i) => `type-${i}`),
+    },
+  },
+})
+
+export default customTwMerge
```

Comment:
This is very cool, and looks like it should solve the issue. one option is to export as `twMerge` then it would be easily replaced in all components just by adjusting the `import` statements on each component.

Reply:
![image](https://github.com/user-attachments/assets/d34432c9-d3f0-421d-bc32-fe27441d0aca)

It totally slipped my mind that calling it `twMerge` would make things sooo much easier when changing the import (especially if we want to utilize this elsewhere oin the future). I'll go ahead and get the export updated for the twMergeConfig and import updated for the headings!